InternetShop/InternetShop/Controllers/UserController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Npgsql;
+using System.Collections.Generic;
+using InternetShop.Controllers;
+
+namespace InternetShop.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class UserController : ControllerBase
+	{
+        [HttpGet(Name = "GetAll")]
+        public async Task<IActionResult> GetAll()
+        {
+            var cs = "Host=localhost;port=3306;Username=postgres;Password=root;Database=dvdrental";
+            using var connection = new NpgsqlConnection(cs);
+            connection.Open();
+
+            string sql = "SELECT * FROM actor";
+            using var cmd = new NpgsqlCommand(sql, connection);
+
+            using NpgsqlDataReader reader = await cmd.ExecuteReaderAsync();
+            List<User> UserList = new List<User>();
```

Comment:
Why are you using camel case this (UserList)? when we usually initialize a variable, we write the name of variable with a small letter

Reply:
And you can use var instead of List<User> f.e var userList = new List<User<();

Reply:
done

Reply:
done